packages/exchange/src/adapters/providers.ts

```diff
@@ -11,14 +11,14 @@ export const Providers = {
   },
   [Exchange.Provider.Encryptus]: {
     sell: {
-      fee: 2,
-      min: 2,
-      max: 2,
+      fee: 2.5,
+      min: 2.5,
```

Comment:
@banklesss here is the min / max ADA to buy and sell, for Banxa the minimum is 100 to buy and sell, for Encryptus please check with Kristina.